reanalysis/hail_filter_and_label.py

```diff
@@ -674,48 +679,33 @@ def green_and_new_from_panelapp(
     return green_gene_set_expression, new_gene_set_expression
 
 
-def informed_repartition(
-    matrix: hl.MatrixTable, post_annotation: bool, temporary_path: str
-):
+def informed_repartition(matrix: hl.MatrixTable):
     """
     uses an estimate of row size to inform the repartitioning of a MT
     aiming for a target partition size of ~10MB
     post-annotation rows are estimated at ~5kB
         - a recursive sys.getsizeof-like guess suggested ~140 Bytes :/
-        - writing a row to text was closer to 20kB :/
-    pre-annotation rows are assumed to be substantially smaller
-    throwing 200k rows into pre-annotation blocks, 100k into post
+        - writing a row to text was closer to 20kB
 
     Kat's thread:
     https://discuss.hail.is/t/best-way-to-repartition-heavily-filtered-matrix-tables/2140
 
     :param matrix:
-    :param post_annotation:
-    :param temporary_path:
     :return: repartitioned matrix
     """
 
-    # calculate partitions, falling back to 1 partition if size is too small
+    # calculate partitions, falling back to 100 partitions if size is too small
     current_rows = matrix.count_rows()
-    if post_annotation:
-        partitions = current_rows // 200000 or 1
-    else:
-        partitions = current_rows // 100000 or 1
+    partitions = current_rows // 200000 or 50
```

Comment:
I don't think you'd want to artificially increase the number of partitions if you only have a few rows.

```suggestion
    partitions = current_rows // 200000 or 1
```